mumbleBot.py

```diff
@@ -71,7 +71,25 @@ def __init__(self, args):
             tt.daemon = True
             tt.start()
 
-        self.mumble = pymumble.Mumble(args.host, user=args.user, port=args.port, password=args.password,
+        if args.host:
+            host = args.host
+        else:
+            host = var.config.get("server", "host")
+        if args.port:
+            port = args.port
+        else:
+            port = var.config.getint("server", "port")
+        if args.password:
+            password = args.password
+        else:
+            password = var.config.get("server", "password")
```

Comment:
You need a default if no password is specified.

Reply:
password should become None through config.get and is hopefully fine for pymumble. I do admit to not testing it :P

Reply:
I need to check if `None` and `""` is the same for pymumble.